src/app/containers/PageHandlers/withVariant/index.test.jsx

```diff
@@ -1,105 +1,88 @@
 import React from 'react';
-import { Router, Route } from 'react-router-dom';
-import { render } from '@testing-library/react';
-import { createMemoryHistory } from 'history';
-import WithVariant from '.';
-import { frontPagePath } from '#app/routes/regex';
-
-describe('WithVariant', () => {
-  const Component = () => <h1>This is the BBC.</h1>;
-  const ComponentWithVariantRedirect = WithVariant(Component);
-  const getMatchProps = (service, path = null) => ({
-    path: path || frontPagePath,
-    params: {
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
```

Comment:
Myself and Michael spent a bit of time on these tests. I'm not sure why you've rewritten them and decided to mock a bunch of things and make everything snapshots.

Reply:
Will look into this.